test/system/253-podman-quadlet.bats

```diff
@@ -0,0 +1,552 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Tests generated configurations for systemd.
+#
+
+load helpers
+load helpers.network
+load helpers.registry
+load helpers.systemd
+
+UNIT_FILES=()
+
+function setup() {
+    skip_if_remote "podman quadlet is not implemented for remote setup yet"
+    skip_if_rootless_cgroupsv1 "Can't use --cgroups=split w/ CGv1 (issue 17456, wontfix)"
+    skip_if_journald_unavailable "Needed for RHEL. FIXME: we might be able to re-enable a subset of tests."
+
+    test -x "$QUADLET" || die "Cannot run quadlet tests without executable \$QUADLET ($QUADLET)"
+
+    basic_setup
+    # Clean all quadlets
+    run_podman quadlet rm --all -f
+}
+
+function teardown() {
+    for UNIT_FILE in ${UNIT_FILES[@]}; do
```

Comment:
Maybe I'm missing something, but I don't see where values are added to `UNIT_FILES`

Reply:
This part is not needed.